packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -14,12 +14,30 @@ import { __, _n, sprintf } from '@wordpress/i18n';
  * @param {boolean} isLast        This is the last block.
  * @param {number}  dir           Direction of movement (> 0 is considered to be going
  *                                 down, < 0 is up).
+ * @param {string} orientation    The orientation of the block movers, vertical or horizontal.
+ * @param {boolean}   isRTL   	  True if current writing system is right to left.
  *
  * @return {string} Label for the block movement controls.
  */
-export function getBlockMoverDescription( selectedCount, type, firstIndex, isFirst, isLast, dir ) {
+export function getBlockMoverDescription( selectedCount, type, firstIndex, isFirst, isLast, dir, orientation, isRTL ) {
 	const position = ( firstIndex + 1 );
 
+	const getMovementDirection = ( mdir ) => {
+		switch ( mdir ) {
+			case 'left':
+				mdir = isRTL ? 'right' : mdir;
+				break;
+
+			case 'right':
+				mdir = isRTL ? 'left' : mdir;
+				break;
+
+			default:
+				break;
+		}
+		return mdir;
+	};
```

Comment:
Same again here about naming the parameter to `moveDirection` which accepts `'up'` or `'down'` and using an `if` instead of a `switch`.